v03/governance/01-raise-validators-number/README.md

```diff
@@ -0,0 +1,10 @@
+Query active proposals:
+
+`quasarnoded q gov proposals`
```

Comment:
need to provide other flag in the commands. like --node , --chain-id . It is always good to provide full commands.